deployments/agw-srs-aws-scenario.yaml

```diff
@@ -0,0 +1,290 @@
+# TODO: Review whole template.
```

Comment:
As we discussed during the engineering sprint, there's the Terraform provider for Juju that can be used to achieve this. I don't think we want to maintain a Cloud Formation template.

Reply:
I agree. I'm already working on it. However, in the meantime I think is worth to keep the CloudFormation one. For testing srs charm and snap, it's super slow to create the full environment manually. When the terraform is done, we can delete this one. What do you think?

Reply:
I agree with the problem: "deploying all magma components and the required infra takes a lot of time". But not with the the solution.

Reply:
Yes I think we should spend our effort on the Terraform provider for Juju.

Reply:
Nice, I'm working on it.

Reply:
Fyi here's the doc, not sure if you saw it: https://registry.terraform.io/providers/juju/juju/latest/docs

Reply:
Yeah I did, thanks anyway!